### ISSUE TITLE
Undefined is not an object (evaluating '_react2.PropTypes.bool')

### ISSUE THREAD

**Issue by iloo123** [moschan/react-native-pull-to-refresh#22]
This error is not fixed while running on iOS. Android had the same issue but i fixed by importing PorpTypes from 'prop-types' instead of importing it from React. Please Help me out. Thanks

**Comment by PanRada** [comment]
+1 same temprarly fix is comment all proptypes in or replace 
import React, { PropTypes } from "react";
wuth
import React from "react";
import PropTypes  from "prop-types";

**Comment by Garib-ka-code** [comment]
still I am getting same error..

**Comment by smaclell** [comment]
I believe this should be fixed by: https://github.com/moschan/react-native-pull-to-refresh/pull/24

**Comment by Garib-ka-code** [comment]
Yes, When I change import React, { PropTypes } from "react" to import PropTypes from "prop-types".
In node_modules then it works. Please update your repo.

**Comment by moschan** [comment]
Hi guys, sorry for late response,

I merged PR and bumped it.

Use `v2.1.3`. Thanks 😸

**Comment by smaclell** [comment]
@moschan thanks for merging the fix!
